pkg/client/policy.go

```diff
@@ -12,9 +11,9 @@ func (c *client) ListPolicies(ctx context.Context) ([]shared.Policy, error) {
 	pageSize := float64(100)
 	pageToken := ""
 	for {
-		resp, err := c.sdk.Policies.List(ctx, operations.C1APIPolicyV1PoliciesListRequest{
-			PageToken: &pageToken,
+		resp, err := c.sdk.PolicySearch.Search(ctx, &shared.SearchPoliciesRequest{
```

Comment:
I'm glad our type system and API responses are so regular. Even though this changed from a C1APIPolicyV1PoliciesListResponse to a C1APIPolicyV1PolicySearchSearchResponse, it has the same resp.ListPolicyResponse.List.